src/Components/FilmPage/FilmPage.js

```diff
@@ -0,0 +1,100 @@
+import {useParams} from 'react-router-dom'
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+import style from '../../styles/FilmPage.module.css';
+import Rating from './Rating'
+
+const FilmPage = () => {
+  const {id} = useParams()
+
+  const [loading,setLoading] = useState(false)
+  const [filmInfo,setFilmInfo] = useState({})
+  const [ratings, setRatings] = useState([])
+
+  useEffect(() => {
+
+    const getFilmInfo = async () => {
+      setLoading(true)
+      axios.get(`http://www.omdbapi.com/?i=${id}&apikey=fb59bcb5`)
+      .then(response => {
+        setFilmInfo(response.data)
+        setRatings(response.data.Ratings)
+      })
+      .catch(error => console.log(error.message))
+    
+      setLoading(false)
+    }
```

Comment:
same as for `getFilms()` function. Read above

Reply:
done